### PR TITLE
'bin/wasabi.sh test:module-name,another-module' will run unit tests f…

### PR DESCRIPTION
**Pull request by shoeffner**
…or those modules and only compile what's necessary for it

### CODE REVIEW

**Comment by felixgao** [comment]
does the shell script works within the submodules?

File: bin/wasabi.sh [hunk 1]
```diff
@@ -282,6 +282,8 @@ for command in ${@:$OPTIND}; do
     start) command="start:cassandra,mysql,wasabi";&
     start:*) commands=$(echo ${command} | cut -d ':' -f 2)
       (IFS=','; for command in ${commands}; do start ${command}; done);;
+    test:*) commands=$(echo ${command} | cut -d ':' -f 2)
```
Comment (felixgao) [inline bin/wasabi.sh]: also seems you need to add that change to the prompt for test subcommand

**Comment by shoeffner** [comment]
You just call it as you always do: from root call bin/wasabi.sh test:experiment-objects for example. It won't work when invoked from modules/experiment-objects via ../../bin/wasabi.sh - if I understand your question correctly.

File: bin/wasabi.sh (hunk 1 above)
Comment (shoeffner) [inline bin/wasabi.sh]: Added it to the usage with [7092306](https://github.com/intuit/wasabi/pull/74/commits/7092306ea567b3866c8652cf9e5b6f2cf071565f)

**Comment by felixgao** [comment]
:shipit:

**Comment by jwtodd** [comment]
color me : impressed @shoeffner  :)

:shipit:

**Pull request merged**